### PR TITLE
cardano-api(gen,test): Remove unnecessary function; Reformat imports

### PR DESCRIPTION
**Pull request by cblp**
No need to convert a Hedgehog group to a Tasty one since Tasty can incorporate Hedgehog properties directly

### CODE REVIEW

File: cardano-api/test/Test/Cardano/Api/Json.hs [hunk 1]
```diff
@@ -9,18 +9,18 @@ module Test.Cardano.Api.Json
 
 import           Cardano.Prelude
 
-import           Cardano.Api
```
Comment (Jimbo4350) [inline cardano-api/test/Test/Cardano/Api/Json.hs:12]: `Cardano.Api` imports should be at the top at it was.

File: cardano-api/test/Test/Cardano/Api/KeysByron.hs [hunk 2]
```diff
@@ -5,15 +5,17 @@ module Test.Cardano.Api.KeysByron
   ( tests
   ) where
 
-import           Cardano.Api
```
Comment (Jimbo4350) [inline cardano-api/test/Test/Cardano/Api/KeysByron.hs:8]: Same here

File: cardano-api/test/Test/Cardano/Api/Metadata.hs [hunk 3]
```diff
@@ -6,16 +6,18 @@ module Test.Cardano.Api.Metadata
   , genTxMetadataValue
   ) where
 
-import           Cardano.Api
```
Comment (Jimbo4350) [inline cardano-api/test/Test/Cardano/Api/Metadata.hs:9]: Same here.

File: cardano-api/test/Test/Cardano/Api/Typed/Address.hs [hunk 4]
```diff
@@ -5,15 +5,17 @@ module Test.Cardano.Api.Typed.Address
   ( tests
   ) where
 
-import           Cardano.Api
```
Comment (Jimbo4350) [inline cardano-api/test/Test/Cardano/Api/Typed/Address.hs:8]: Same here

File: cardano-api/test/Test/Cardano/Api/Typed/Bech32.hs [hunk 5]
```diff
@@ -3,12 +3,14 @@ module Test.Cardano.Api.Typed.Bech32
...
+import           Hedgehog (Property)
+import           Test.Tasty (TestTree)
+import           Test.Tasty.Hedgehog (testProperty)
+import           Test.Tasty.TH (testGroupGenerator)
+
 import           Cardano.Api
 import           Gen.Cardano.Api.Typed
 import           Gen.Hedgehog.Roundtrip.Bech32 (roundtrip_Bech32)
-import           Gen.Tasty.Hedgehog.Group (fromGroup)
-import           Hedgehog (Property, discover)
```
Comment (Jimbo4350) [inline cardano-api/test/Test/Cardano/Api/Typed/Bech32.hs:10]: Generally we leave the test modules last.

File: cardano-api/test/Test/Cardano/Api/Typed/Envelope.hs [hunk 6]
```diff
@@ -6,15 +6,17 @@ module Test.Cardano.Api.Typed.Envelope
   ( tests
   ) where
 
-import           Cardano.Api
```
Comment (Jimbo4350) [inline cardano-api/test/Test/Cardano/Api/Typed/Envelope.hs:9]: Same here

File: cardano-api/test/Test/Cardano/Api/Typed/Ord.hs [hunk 7]
```diff
@@ -4,16 +4,18 @@ module Test.Cardano.Api.Typed.Ord
   ( tests
   ) where
 
-import           Cardano.Api
```
Comment (Jimbo4350) [inline cardano-api/test/Test/Cardano/Api/Typed/Ord.hs:7]: Same here

File: cardano-api/test/Test/Cardano/Api/Typed/RawBytes.hs [hunk 8]
```diff
@@ -5,15 +5,17 @@ module Test.Cardano.Api.Typed.RawBytes
   ( tests
   ) where
 
-import           Cardano.Api
```
Comment (Jimbo4350) [inline cardano-api/test/Test/Cardano/Api/Typed/RawBytes.hs:8]: Same here

File: cardano-api/test/Test/Cardano/Api/Typed/Script.hs [hunk 9]
```diff
@@ -4,17 +4,19 @@ module Test.Cardano.Api.Typed.Script
   ( tests
   ) where
 
-import           Cardano.Api
```
Comment (Jimbo4350) [inline cardano-api/test/Test/Cardano/Api/Typed/Script.hs:7]: Same here.

File: cardano-api/test/Test/Cardano/Api/Typed/Value.hs [hunk 10]
```diff
@@ -4,16 +4,18 @@ module Test.Cardano.Api.Typed.Value
   ( tests
   ) where
 
-import           Cardano.Api.Shelley
```
Comment (Jimbo4350) [inline cardano-api/test/Test/Cardano/Api/Typed/Value.hs:7]: Same here

**Review by Jimbo4350 — CHANGES_REQUESTED** [review]

File: cardano-api/test/Test/Cardano/Api/Json.hs (hunk 1 above)
Comment (Jimbo4350) [reply cardano-api/test/Test/Cardano/Api/Json.hs:12]: You're right, I just had a look in `cardano-api`

**Comment by cblp** [comment]
bors merge

**Pull request merged**